### PR TITLE
tickets/DM-29333

### PR DESCRIPTION
**Pull request by mareuter**

### CODE REVIEW

File: ncsa_teststand/lsst-teststand-ts1/shutdown_love [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /deploy-lsstts/.love_secrets
```
Comment (wvreeven) [inline ncsa_teststand/lsst-teststand-ts1/shutdown_love:2]: You may want to stick to a common naming convention. Here the `.love_secrets` file doesn't have an extension but in the amos01 script it does.

File: summit/amor01/shutdown_love [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /deploy-lsstts/.love-cred.sh
```
Comment (wvreeven) [inline summit/amor01/shutdown_love:2]: You may want to stick to a common naming convention. Here the `.love_secrets` file does have an extension but in the lsst-teststand-ts1 script it doesn't.

**Review by wvreeven — APPROVED** [review]
Looks good to me but please try to stick to one naming convention regarding file extensions.

File: summit/amor01/shutdown_love (hunk 2 above)
Comment (mareuter) [reply summit/amor01/shutdown_love:2]: OK. I'll make the file the same between the sites and adjust the scripts accordingly.

**Pull request merged**